scraper/src/config/browser_handler.py

```diff
@@ -10,7 +10,7 @@
 class BrowserHandler:
     @staticmethod
     def conf_need_browser(config_original_content, js_render):
-        group_regex = re.compile("\\(\?P<(.+?)>.+?\\)")
+        group_regex = re.compile(r'\(\?P<(.+?)>.+?\)')
```

Comment:
if you use the regex often, do not compile it within the function, but store this in an external (const global) variable to compile it once. Otherwise, you can pass the string pattern directly to `re.findall`

Reply:
Will keep it in our backlog. Thanks for sharing (not the PR's topic)